EIPS/eip-draft_disable_selfdestruct.md

```diff
@@ -35,8 +35,8 @@ will reduce complexity of EVM going forward, disable GasToken2 (but not GasToken
 polymorthic contracts impossible.
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
-After certain block number, the semantics of `SELFDESTRUCT` becomes the same as the combination of `POP` followed by `STOP`. Gas cost is the same as the gas cost
-of `POP`, which is 2 gas. No value transfer occurs and no gas refund is given.
+After certain block number, the semantics of `SELFDESTRUCT` becomes the same as the combination of `POP`, followed by transferring remaining ETH
+to the address popped from the stack, followed by `STOP`. Gas cost is the same as the gas cost of non-zero value transfer, which is 9000 gas. No gas refund is given.
```

Comment:
This gas scheme will make it cheaper to create new account via `SELFDESTRUCT` (which costs 30000 right now). Better either:

* Keep the current `SELFDESTRUCT` gas cost: `5000 + 25000 if new account`.
* Or, increase the base to `9000` like this but also with the 25000 new account penalty: `9000 + 25000 if new account`.

Also note that `9000` comes from `CALL` opcode, but `SELFDESTRUCT` is not exactly the same as `CALL` transfer because the former will never execute the target if it's a contract.

Reply:
It's currently cheapest to create a new account with a regular 21000 gwei transfer

Reply:
It's currently cheapest to create an account using a regular 21000 transfer